src/checkbox/index.tsx

```diff
@@ -9,7 +9,7 @@ import useBaseComponent from '../internal/hooks/use-base-component';
 export { CheckboxProps };
 
 const Checkbox = React.forwardRef(({ ...props }: CheckboxProps, ref: React.Ref<CheckboxProps.Ref>) => {
-  const baseComponentProps = useBaseComponent('Checkbox');
+  const baseComponentProps = useBaseComponent('Checkbox', { disabled: props.disabled, checked: props.checked });
```

Comment:
Why is there no `name` prop? Are they going to distinguish individual controls using contexts?